components/kubeflow/pytorch-launcher/component.yaml

```diff
@@ -0,0 +1,50 @@
+name: Kubeflow - Launch PyTorchJob
+description: Kubeflow PyTorchJob launcher
+inputs:
+- {name: name,                       type: String,                       description: 'PyTorchJob name.'}
```

Comment:
Is the name needed? Can the name be generated (`generateName`)?

Reply:
Yeah good question.  I followed the tf launcher's lead, but personally found it annoying having to manually change the name sometimes when testing.

Would a generated name always be desired, or would there be a business case for sometimes being able to get a deterministic name?  I'm thinking in the case of a pipeline, I might want to name it something like "pytorchjob-train-model-x-RUNID" or something.  Or is there a better way to deterministically grab the pytorch job someone just launched?

We could have an optional name argument, where if specified then you get that name exactly but if left blank you get `generateName: pytorch-` in the yaml?  Or as overkill, could also have an optional `generateName` arg that lets you specify the pattern (`--generateName my-pytorch-`).

I'm open for whatever

Reply:
>Would a generated name always be desired, or would there be a business case for sometimes being able to get a deterministic name?

I'm not sure. For me a generated name is fine. But I also might not be the target user of these components.

Perhaps we can start with not exposing the name and see whether people find it sufficient.